We need LDFLAGS to be respected otherwise there are QA warnings as the right
flags don't get used.

Upstream-Status: Inappropriate [oe-core specific]
Signed-off-by: Ross Burton <ross.burton@arm.com>

From d92af0483c20365fd0af740d0baef8870b4aa374 Mon Sep 17 00:00:00 2001
From: Abdur Rehman <abdur_rehman@mentor.com>
Date: Wed, 26 Aug 2015 19:18:11 +0500
Subject: [PATCH] tcf-agent: obey LDFLAGS

Signed-off-by: Abdur Rehman <abdur_rehman@mentor.com>

Rebased to 4a2c4baaccbc8c29ce0297705de9a4e096d57ce5 version

Signed-off-by: Mark hatle <mark.hatle@amd.com>


---
 Makefile.inc |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

Index: agent/Makefile.inc
===================================================================
--- agent.orig/Makefile.inc
+++ agent/Makefile.inc
@@ -111,7 +111,7 @@ NO_LINK_F   ?= -c
 # Linker definition and flags
 
 LINK        ?= $(CC)
-LINK_FLAGS  ?= $(CFLAGS)
+LINK_FLAGS  ?= $(LDFLAGS) $(CFLAGS)
 LINK_OUT_F  ?= $(OUT_OBJ_F)
 
 # Archiver definition and flags
